test/hga/HGA_Regression.ipynb

```diff
@@ -1458,7 +1451,7 @@
    "source": [
     "### Sentinel test case 13:\n",
     "\n",
-    "This test requests a bounding box spatial subset while not specifying any variable subset, so all variables should be returned. The Sentinel data has 7 variables with two different sets of grid sizes.  This leads to a request that cannot be fulfilled by HGA and an exception should be raised.\n"
+    "The Sentinel data has 7 variables with two different sets of grid sizes. This test requests a bounding box spatial subset while specifying two variables with different grids. This leads to a request that cannot be fulfilled by HGA and an exception should be raised. The internal exception type is an `IncompatibleVariableError`, which can be identified via the string reported to the end-user."
```

Comment:
This update to the test seems more in line with what it was meant to be doing. The previous failure was baulking because the `productBoundingBox` variables is basically a string, and is incompatible with HGA. I'm not sure how long NetCDF-4 handling will persist in this service, so it seemed prudent in the short term to adjust the test. If NetCDF-4 files are handled more in the future, we should add a newer test to cover the incompatible variable processing (and also decide the behaviour we want).

Reply:
Also, I experienced issues with my local Harmony instance with this test. A version from a week or so ago wasn't correctly propagating error messages from the service container out to the end-user visible results. Pulling the latest version of the `main` branch of the Harmony repo and rebuilding all three Harmony images fixed that up, though.